pathology/nuclick/nuclei_classification_training_notebook.ipynb

```diff
@@ -213,7 +179,7 @@
    "source": [
     "## Pre-processing utility functions\n",
     "\n",
-    "`consep_nuclei_dataset` reads the raw Image and Matlab files provided in ConSeP dataset.  For each Nuclei it tries to create a patch of 128x128 with single nuclei labeled with correspnding class index and rest of the nuclei falling in this patch are labeled as others (mask_value: 255)\n",
+    "`consep_nuclei_dataset` reads the raw Image and Matlab files provided in CoNSeP dataset.  For each Nuclei it tries to create a patch of 128x128 with single nuclei labeled with correspnding class index and rest of the nuclei falling in this patch are labeled as others (mask_value: 255)\n",
```

Comment:
The consep_nuclei_dataset function processes the raw image and Matlab files from the CoNSeP dataset. For each nucleus, it generates a 128x128 patch wherein the target nucleus is labeled with its corresponding class index, and all other nuclei within the patch are labeled as 'others' (using a mask value of 255).